cypress/component/ProgressBar.cy.tsx

```diff
@@ -88,13 +89,18 @@ describe("ProgressBar", () => {
       .and("to.have.string", "ProgressBar")
       .should("to.have.string", "multistep")
 
-    cy.get('[data-id="stepIndicator"]').should("have.length", multiSteProps["totalSteps"]);
-    // cy.get('[data-id="stepIndicatorFill"]').eq(multiSteProps["currentStep"]).should("have.attr", "class")
-    //   .and("to.have.string", "isActive")
-    // cy.get('[data-id="stepIndicatorFill"]').eq(multiSteProps["currentStep"]).should("have.css", "width", `${multiSteProps["stepProgress"]}%`);
-  });
-
+    cy.get(stepIndicator)
+      .should("have.length", multiSteProps["totalSteps"]);
```

Comment:
Did you mean `multiStepProps` with 2 `p`s?

Reply:
saving in letters? 😆 fixing it